src/test/java/pl/targosz/homework/Lesson7Task6/CurrencyCalculatorTest.java

```diff
@@ -0,0 +1,31 @@
+package pl.targosz.homework.Lesson7Task6;
+
+
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import java.math.BigDecimal;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.when;
+
+class CurrencyCalculatorTest {
+    @Test
+    void shouldReturnTheSellAmountFromGBPToPLN() {
+        CurrencyInterface currencyInterface = Mockito.mock(CurrencyInterface.class);
+        CurrencyCalculator currencyCalculator = new CurrencyCalculator(currencyInterface);
+
+        when(currencyCalculator.sellRateCalculator(BigDecimal.valueOf(135.456),Currency.GBP)).thenReturn(BigDecimal.valueOf(135.456*5.42));
```

Comment:
Ale czemu mokujesz cały currencyCalculator, a nie CurrencyInterface ? 

Aktualnie nie testujesz w ogóle swojego kalkulatora. Mówisz Mockito: "Weź mój currencyCalculator i zrób tak, że jak ktoś zawoła go z argumentem x,y to niech zwróci z".

A chodzi o to, aby zamockować CurrencyInterface, bo chcesz sprawdzić czy twój kalkulator działa (licząc, że CurrencyInterface  będzie dzialał tak jak zakładasz)